pkg/bindfilter/bind_filter_test.go

```diff
@@ -0,0 +1,211 @@
+//go:build windows
+// +build windows
+
+package bindfilter
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func TestApplyFileBinding(t *testing.T) {
```

Comment:
It just occured to me, for testing we sometimes pull the OS version from the registry and use that to skip tests: https://github.com/containerd/containerd/blob/97480afdac09c947d48f5e3a134db86c78f4bfa6/pkg/os/os_windows_test.go#L40-L55

It may be cleaner than relying on skipping tests in the ci file

Reply:
We'd need to pull in hcsshim as a dependency for the osversion package, or duplicate the values in ```hcsshim/osversion/windowsbuilds.go```, here. Which option would you prefer? :smile:

Reply:
The later: you can porbably just compare against `17763` instead of copying in all the build numbers
(we probably should move the OS versions from hcsshim to here...)

Reply:
Done. PTAL.